gcp_variant_transforms/bq_to_vcf.py

```diff
@@ -153,6 +151,26 @@ def _bigquery_to_vcf_shards(
          | vcfio.WriteVcfDataLines())
 
 
+def _form_customized_query(known_args):
+  # type: (argparse.Namespace) -> str
+  """Returns a customized query for the interested regions."""
+  base_query = _BASE_QUERY_TEMPLATE.format(INPUT_TABLE='.'.join(
+      bigquery_util.parse_table_reference(known_args.input_table)))
+  if not known_args.genomic_regions:
```

Comment:
nit: rather than this, I suggest to just proceed with making conditions list and only append `WHERE` if conditions is set. It's more generic that way and you dont need to worry about updating this if when we add more customized logic (e.g. info/format/call filters).